design.css

```diff
@@ -0,0 +1,41 @@
+body{
+    margin-top: 0px;
+     margin-right: 0px;
+     margin-bottom: 0px;
+    margin-left: 0px;
+}
+ul{
+    padding-inline-start: 1px;
+    position:absolutes;
+    float :left;
```

Comment:
: should stick with the attribute and not with the value.  Ideally you need to leave a space after colon.

Reply:
made the those changes